requirements_pip.txt

```diff
@@ -0,0 +1,2 @@
+# python library that contain numpy and scipy already
+ase==3.19.0
```

Comment:
We should be more explicit on the dependencies. If we import numpy directly ourself, it should be in the requirements.txt file. Same for scipy, and any other dependency.